lib/LMSManagers/LMSEventManager.php

```diff
@@ -392,6 +402,9 @@ public function GetEventList(array $params)
             $closedfilter = '';
         }
 
+        $netdevfilter = empty($netdevid) ? null : ' AND events.netdevid = ' . intval($netdevid);
+        $netnodefilter = empty($netnodeid) ? null : ' AND events.netnodeid = ' . intval($netnodeid);
+
```

Comment:
Sklejanie stringa z null-em nie generuje ostrzeżeń PHP (sklejanie jest niżej).
Bardziej elegancko było zamiast `null` użyć `''`.

Reply:
> Sklejanie stringa z null-em nie generuje ostrzeżeń PHP (sklejanie jest niżej).

Tak potwierdzam, że NIE generuje ostrzeżeń - dlatego tak zrobiłem.

https://github.com/chilek/lms/pull/2273/commits/4fbdf8f3c22cf1b7212d2126cedcedcd6b7c28ab
Napisz mi proszę tylko dlaczego bardziej elegancko jest używać `''` zamiast `null`.

Reply:
Widzę też że operator trójstronny pozwala na skrócenie zapisu do np. takiej formy:
`$netnodefilter = empty($netnodeid) ?: ' AND events.netnodeid = ' . intval($netnodeid);`

Jak chcesz mieć w kodzie?

Reply:
Poczytaj do czego służy `?:`.

Reply:
![image](https://user-images.githubusercontent.com/17087236/183139356-e526febf-fec6-469c-b0c9-b0f766fab932.png)

Może mylisz z innym językiem programowania w którym ?: znaczy coś innego?

Reply:
Nawet zrobiłem https://github.com/chilek/lms/pull/2306, żeby zunifikować kod w całym LMS w tym zakresie.

Reply:
https://github.com/chilek/lms/pull/2273/commits/d9b8111fed74a2dcdce2bf32471f5c511348d9f2